File: datasets/IR-Plag/convertai-original-typescript-rest-java/case-06/plagiarized/L6/05/singlearray.java
package IR

import java.util.*;

class singlearray
{
	static int[] n = new int[10];

	public static void main (String[] args)
	{
				
		Scanner inp = new Scanner(System.in);

		int i=9;
		int a=0;
		while (i >=0) 
		{
			System.out.print("Read a number: ");
			n[a] = inp.nextInt();
			a++;
			i--;
		}
	print();
	}
		static void print()
	{
				for (int a = 0; a <=9; a++) 
		{
			System.out.println(n[9-a]);
		}
	}
}

File: datasets/IR-Plag/original-java-rest-kotlin/case-06/plagiarized/L4/05/singlearray.kt
package IR

import java.util.*

internal object singlearray {
    var n: IntArray = IntArray(10)
    fun print() {
        for (a in 9 downTo 0) {
            println(n[a])
        }
    }

    @JvmStatic
    fun main(args: Array<String>) {
        val inp = Scanner(System.`in`)

        for (i in 0..9) {
            print("Read a number: ")
            n[i] = inp.nextInt()
        }
        print()
    }
}